### ISSUE TITLE
Name does not fit for phpBB 4.0 & composer 2.0

### ISSUE THREAD

**Issue by Crizz0** [rxu/posts_merging#45]
Current name:
> "name": "rxu/PostsMerging",

Composer 1.9.x warns with:
`Deprecation warning: require.rxu/PostsMerging is invalid, it should not contain uppercase characters. Please use rxu/postsmerging instead. Make sure you fix this as Composer 2.0 will error.`

phpBB is following these rules with the next major release: https://www.phpbb.com/community/viewtopic.php?f=461&t=2549901

**Comment by rxu** [comment]
Yeah thanks, I'm aware of it. The problem in changing the namespace with the next release is that the policy says
> It won't be possible to migrate any data from your current extension that uses Vendor/Ext to your phpBB 4.0 compatible extension.

I need to find a way to migrate from one extension to another smoothly.

**Comment by Crizz0** [comment]
For this extension it wouldn't be to hard to:
1. uninstall and delete it completely
2. install fresh version
3. restore the settings of this extension
because there are only 3 options and 1 text field. 

But yes, this is not directly smooth :)

**Comment by rxu** [comment]
The ext also adds a column `post_created` to the posts table which has to be handled somehow.
I'm also thinking of adding own extension table for this instead.

**Comment by Crizz0** [comment]
Ah, I remember. That new column made my installation with 1.2 M posts take a while. 👼

**Comment by rxu** [comment]
For some installations it even throws SQL error sometimes as it can't alter the table to add the column. That's a common SQL problem. #42 btw right next to this issue :)
Anyway, this is the break point for changing the ext name to lowercased here. But I guess I'll do it next release anyway.

**Comment by rxu** [comment]
I guess it's fixed now :)

**Comment by Crizz0** [comment]
Will this migration run on a fresh install: https://github.com/rxu/posts_merging/blob/develop-3.2/migrations/v_3_0_0.php ?

**Comment by rxu** [comment]
Yes, it should.
